src/lib/markdown.ts

```diff
@@ -1,28 +1,42 @@
-import { marked } from 'marked'
+import { Marked } from 'marked'
 import DOMPurify from 'dompurify'
 
+interface IImageParams {
+  _href: string
+  _title: string
+  _text: string
+}
+
+interface ILinkParams {
+  href: string
+  _title: string
+  text: string
+}
+
 // The U+2028 character (LINE SEPARATOR) is sometimes used as a line break, but it is treated as a space in some web environments,
 // causing unexpected rendering issues. To avoid this, it's recommended to replace it with a standard line break character such as `\n`.
-const LINE_SEPARATOR = /\u2028/g
+const LINE_SEPARATOR: RegExp = /\u2028/g
 
 // LINE_BREAK_VISUAL is used for formatting when the message is displayed as a single line in a preview.
 // It replaces line breaks with a visual symbol (↵) to indicate where new lines exist in the original text.
-const LINE_BREAK_VISUAL = '↵ '
+const LINE_BREAK_VISUAL: string = '↵ '
+
+const markedInstance = new Marked()
 
-marked.setOptions({
+markedInstance.setOptions({
   // marked sanitize is deprecated, using DOMPurify
   // sanitize: true,
   gfm: true,
   breaks: true
 })
 
-const renderer = new marked.Renderer()
+const renderer = new markedInstance.Renderer()
 
-renderer.image = function ({ _href, _title, _text }) {
+renderer.image = function ({ _href, _title, _text }: IImageParams): string {
```

Comment:
This may also work with Type Inference. You don't need to define custom types (IImageParams, ILinkParams).